parent/pom.xml

```diff
@@ -153,6 +153,8 @@
 		<!-- discovery.tpc.version usually is usually set to ${tpc.version} but it
 		can happen that they need to have different values, hence 2 properties -->
 		<discovery.tpc.version>${TARGET_PLATFORM_CENTRAL_MAX}</discovery.tpc.version>
+		<testProduct>com.jboss.devstudio.core.product</testProduct>
+		<devstudio.repository>https://devstudio.redhat.com/10.0/snapshots/updates/</devstudio.repository>
```

Comment:
Usually we want to run our tests against nighly build so we are using /snapshots/ site as default. If we want to run against stable or some other specific build we can override the property. 

Plan is to move all integration-tests to component specific repository which means there will be no jbosstools-integration-tests repo and no jbosstools-integration-tests pom. That is why we need to move all this configuration to jbt parent pom.